packages/dd-trace/src/opentracing/propagation/text_map.js

```diff
@@ -359,6 +361,11 @@ class TextMapPropagator {
       })
 
       this._extractBaggageItems(carrier, spanContext)
+
+      if (carrier.hasOwnProperty(googleCloudTraceContextKey) && !carrier.hasOwnProperty(traceKey)) {
```

Comment:
Google sets `traceparent` and `x-cloud-trace-context` headers when our cloud function is triggered, used for Google Cloud Tracing. Because of the is, the tracer sets the `parent_id` of our cloud function trace's root span, thereby causing the span to not show up under the `Trace Root Spans` in Datadog.

I've talked with my team, and we thought it would make sense to drop the span context span_id, in the case that datadog headers aren't set

Reply:
This would defeat the purpose of W3C support. It's also out of the scope of the responsibilities of the propagator.